lua/plenary/path.lua

```diff
@@ -202,20 +202,31 @@ function Path:mkdir(opts)
   opts = opts or {}
 
   local mode = opts.mode or 448 -- 0700 -> decimal
-  local parents = F.if_nil(opts.parents, false)
-  local exists_ok = F.if_nil(opts.exists_ok, true)
+  local parents = F.if_nil(opts.parents, false, opts.parents)
+  local exists_ok = F.if_nil(opts.exists_ok, true, opts.exists_ok)
```

Comment:
I think that requires a third argument?

Reply:
not sure but isn't `parents = opts.parents == nil and false or true better? then if_nil`

Reply:
no, because that won't work.

If the first result is `falsey` , it will never be selected. That's why I created `if_nil`

Reply:
oh make sense